src/components/header/Header.jsx

```diff
@@ -32,6 +40,30 @@ export default class Header extends React.Component {
             userWidget = (
                 <UserMenu user={ userData }/>
             );
+
+            let callAssignments = this.props.assignments.getIn(['assignmentList', 'items']);
+
+            if (callAssignments.size > 0) {
```

Comment:
`callAssignments` can be null if they have not yet been loaded. Needs a check for null conditions:

```
if (callAssignments && callAssignments.size > 0) { //...
```